cmd/sovereignnode/dataCodec/errors.go

```diff
@@ -0,0 +1,9 @@
+package dataCodec
+
+import "errors"
+
+var errEmptyData = errors.New("empty data provided")
```

Comment:
I think we already have this error defined.
Either use that one, or rename it to actually be more descriptive like (empty bytes to deserialize event data from)